### PR TITLE
Fix batchinserter startup error

### PR DESCRIPTION
**Pull request by ragadeeshu**

### CODE REVIEW

File: integrationtests/src/test/java/org/neo4j/unsafe/batchinsert/BatchInsertersIT.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+import static org.neo4j.unsafe.batchinsert.BatchInserters.inserter;
+
+public class BatchInsertersIT
+{
+
+    @Rule
+    public TestDirectory testDirectory = TestDirectory.testDirectory();
+    @Rule
+    public EphemeralFileSystemRule fileSystemRule = new EphemeralFileSystemRule();
```
Comment (chrisvest) [inline integrationtests/src/test/java/org/neo4j/unsafe/batchinsert/BatchInsertersIT.java]: You're not using this file system rule.

File: integrationtests/src/test/java/org/neo4j/unsafe/batchinsert/BatchInsertersIT.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    public void shouldStartBatchInserterWithRealIndexProvider() throws Exception
+    {
+        BatchInserter inserter = inserter( testDirectory.graphDbDir(), MapUtil.stringMap(), getKernelExtensions() );
+        //If we didn't throw, all good!
+        inserter.shutdown();
+    }
+
+    private Iterable<KernelExtensionFactory<?>> getKernelExtensions()
+    {
+        return Iterables.asIterable( new NativeLuceneFusionIndexProviderFactory20() );
```
Comment (chrisvest) [inline integrationtests/src/test/java/org/neo4j/unsafe/batchinsert/BatchInsertersIT.java:52]: What if we service load the kernel extensions, so we have more than just this one involved? That's the default behaviour when building a `BatchInserter`.

**Review by chrisvest — APPROVED** [review]
Good, but you could improve the testing.

File: integrationtests/src/test/java/org/neo4j/unsafe/batchinsert/BatchInsertersIT.java (hunk 1 above)
Comment (ragadeeshu) [reply integrationtests/src/test/java/org/neo4j/unsafe/batchinsert/BatchInsertersIT.java]: Oops

File: integrationtests/src/test/java/org/neo4j/unsafe/batchinsert/BatchInsertersIT.java (hunk 2 above)
Comment (ragadeeshu) [reply integrationtests/src/test/java/org/neo4j/unsafe/batchinsert/BatchInsertersIT.java:52]: @chrisvest If you know of a place to put this test where no test kernel extensions are available, I'd be more than happy to put it there! :)

In the current form, the test versions of kernel extensions will load fine and stop it from crashing, which defeats the purpose of this test.

**Review by klaren — APPROVED** [review]

File: integrationtests/src/test/java/org/neo4j/unsafe/batchinsert/BatchInsertersIT.java (hunk 2 above)
Comment (chrisvest) [reply integrationtests/src/test/java/org/neo4j/unsafe/batchinsert/BatchInsertersIT.java:52]: Alright, then.

**Pull request merged**